package.json

```diff
@@ -15,7 +15,8 @@
     "test": "jest --passWithNoTests --no-cache"
   },
   "dependencies": {
-    "fs-extra": "^9.0.1"
+    "fs-extra": "^9.0.1",
+    "globby": "^11.0.1"
```

Comment:
I think we dropped this? Or maybe it's just a dev dep now?

Reply:
Yep, I just followed up with a full removal.